layouts/partials/ecosystem/registry/entry.html

```diff
@@ -192,6 +213,7 @@ <h6><i class="fa-solid fa-forward"></i> Quick Install</h6>
             </li>
             {{- end -}}
             </div>
+            <!-- Existing URL links -->
```

Comment:
What is this comment for?

Reply:
@svrnm This one is to indicate a section where URL links related to the registry entry are already defined. This could include links to the repository, documentation, or other relevant resources.

Reply:
Yes, please remove it